### ISSUE TITLE
Quick access to proxy log with a default docker log driver 

### ISSUE THREAD

**Issue by mcctomsk** [vfarcic/docker-flow-proxy#234]
When I try to get container log into file by executing this command:
`docker logs $(docker ps --filter "name=proxy_proxy*" -q) > proxy.log` I get empty file and content of log on screen
```
Docker version 17.04.0-ce, build 4845c56
docker-compose version 1.11.2, build dfed245
docker info | grep 'Logging Driver'
Logging Driver: json-file
```

**Comment by vfarcic** [comment]
The fix is available in the release 1.386. Can you please try it out and let me know if you experience any problem.

**Comment by mcctomsk** [comment]
Awesome. It works as expected.
